operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -261,7 +263,9 @@ runtimeConfig: {}
 
 # RBAC configuration
 rbac:
-  pspEnabled: true
+  create: false
```

Comment:
let's default this to true to have no change to golden record and default behavior